pyqtgraph/widgets/PenSelectorDialogbox.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'PenSelectorDialogbox.ui'
+#
+# Created by: PyQt4 UI code generator 4.11.4
```

Comment:
I know this was generated using PyQt4, but if it had been generated with Qt5 or Qt6, many of the widgets come from QtWidgets, not QtGui.

Reply:
Many of the widgets in pyqtgraph come from `QtGui`... Is this just old code? I thought it was for a reason, but I'm happy to transition to `QtWidgets` instead